core/templates/dev/head/player/ConversationSkinDirective.js

```diff
@@ -441,11 +441,15 @@ oppia.directive('conversationSkin', [function() {
             $rootScope.loadingMessage = '';
             $scope.hasFullyLoaded = true;
 
-            // If the exploration is embedded, use as site language the
-            // exploration language.
+            // If the exploration is embedded, use the exploration language
+            // as site language. If the exploration language is not supported
+            // as site language, English is used as default.
             var siteLanguage = oppiaPlayerService.getExplorationLanguageCode();
-            if ($window.GLOBALS.SUPPORTED_SITE_LANGUAGES && $scope.isIframed) {
+            if ($window.GLOBALS.SUPPORTED_SITE_LANGUAGES[siteLanguage] &&
```

Comment:
Just to double check: previously, you checked for the existence of $window.GLOBALS.SUPPORTED_SITE_LANGUAGES. Was that an error (i.e. is GLOBALS.SUPPORTED_SITE_LANGUAGES always present)? If not, we may still need to check for it here, in order  to prevent a "cannot read siteLanguage of undefined" console error.